apps/alert_processor/test/alert_processor/rules_engine/notification_builder_test.exs

```diff
@@ -83,4 +85,84 @@ defmodule AlertProcessor.NotificationBuilderTest do
       assert %Notification{phone_number: ^expected_phone_number, email: nil} = notification
     end
   end
+
+  describe "Notification.call_to_action" do
+    setup do
+      user = insert(:user)
+      sub = insert(:subscription, user: user)
+
+      {:ok, user_subs: {user, [sub]}}
+    end
+
+    test "includes dotcom url and mbta go CTA if single bus route is effected", %{
+      user_subs: user_subs
+    } do
+      now = naive_to_local(~N[2018-01-11 14:10:55Z])
+      one_hour_ago = DateTime.add(now, -3600)
```

Comment:
(second syntax nitpick) This is just personal preference, but this can also be written DateTime.add(now, -1, :hour)- you can also do DateTime.add(now, 1, :day) for days and so forth.